### DISCUSSION TITLE
Keeping up with changes in this repo?

### DISCUSSION THREAD

**Discussion by robbibt** [fpgmaas/cookiecutter-uv#12] [General]
Hey @fpgmaas, thanks so much for all your work on this repo - as someone without a software engineering background, I've found it super valuable for learning about Python packaging and how the various parts fit together - it's so much easier to learn from something that works vs. starting from scratch!

I've been keeping an eye on some of the recent PRs here, and it got me wondering: what is the best way to keep my downstream repo up-to-date with improvements being made here to the template? In particular, the code here can sometimes be a little tricky to copy across directly as it includes configuration details as well as the output code itself.

Any suggestions or thoughts on how to make sure we don't miss out on any new or improved features here would be fantastic, especially given `uv` itself is evolving rapidly too!

**Comment by jakob1379** [comment]
A common tool for this is to use the tool called `cruft`. It saves your answers so the same setup can be recreated and used for updating existing repos that was initialized. [cruft](https://github.com/cruft/cruft)

You simply use `cruft update` and voila, your project is updated.
